tests/_test_case_base.py

```diff
@@ -0,0 +1,8 @@
+import tornado.testing
+
+
+class BaseAsyncHTTPTestCase(tornado.testing.AsyncHTTPTestCase):
```

Comment:
I believe a pytest convention is to use the `test` prefix for actual test files and fixtures, helpers, and base classes go in `conftest.py` (or maybe your `helpers.py`).  At any rate, these base classes seem a bit unnecessarily spread out.

Reply:
They are but the main reason is to allow conditional imports in older versions of python. I'll try to move to conftest or helpers but I think will still have to split ones with newer python syntax to different files.

Reply:
For sure, but I thought `_handlers_async_py35.py` was the only culprit.  I'm possibly not tracking correctly.